copy.go

```diff
@@ -0,0 +1,46 @@
+package fssync
+
+import (
+	"io"
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+// copyContent is highly inspired from io.Copy, but calls to fadvise have been
+// added to prevent caching the whole content of the files during the process,
+// impacting the whole OS disk cache
+func (s *FsSyncer) copyContent(src, dst *os.File) (written int64, err error) {
```

Comment:
```suggestion
func (s *FsSyncer) copyContent(src, dst *os.File) (int64, error) {
```
I think we agreed it's not a good idea to use this Golang construct

Reply:
It's actually how it's writte in `io.Copy` in the stdlib, ok to change it